docs/builders/docker.mdx

```diff
@@ -340,6 +340,20 @@ of post-processors that are treated as as single pipeline, see
 [Post-Processors](/packer/docs/templates/legacy_json_templates/post-processors) for more information):
 
 <Tabs>
+<Tab heading="HCL2">
+
+```hcl
+  post-processors {
+    post-processor "docker-import" {
+        repository =  "myrepo/myimage"
+        tag = "0.7"
```

Comment:
```suggestion
        tags = ["0.7"]
```

Reply:
The `docker-import` post-processor only accepts a single tag, from the `tag` (`string`) field. The tag post-processor allows multiple with the `tags` (`list[string]`) field or the `tag` (`list[string]`) field (maintained for backward compatibility.